x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.yml

```diff
@@ -434,6 +434,10 @@ processors:
       field: checkpoint.ifdir
       target_field: network.direction
       ignore_missing: true
+  - convert:
+      field: checkpoint.bytes
+      type: long
+      ignore_missing: true
   - rename:
       field: checkpoint.bytes
       target_field: network.bytes
```

Comment:
Can this be merged into a single step ? - https://github.com/elastic/integrations/pull/3800#discussion_r927481295

Reply:
That leaves the value in `checkpoint.bytes`.